brewtils/rest/easy_client.py

```diff
@@ -237,19 +237,65 @@ def get_logging_config(self, system_name=None, local=False):
         return self.client.get_logging_config(local=local)
 
     @wrap_response(
-        parse_method="parse_system", parse_many=False, default_exc=FetchError
+        parse_method="parse_garden", parse_many=False, default_exc=FetchError
     )
-    def get_system(self, system_id, **kwargs):
+    def get_garden(self, garden_name, **kwargs):
         """Get a System
 
+        Args:
+            garden_name: Name of garden to retrieve
+
+        Returns:
+            The Garden
+
+        """
+        return self.client.get_garden(garden_name, **kwargs)
+
+    @wrap_response(parse_method="parse_garden", parse_many=False, default_exc=SaveError)
+    def create_garden(self, garden):
+        """Create a new Garden
+
+        Args:
+            garden (Garden): The Garden to create
+
+        Returns:
+            Garden: The newly-created Garden
+
+        """
+        return self.client.post_gardens(SchemaParser.serialize_garden(garden))
+
+    def remove_garden(self, garden_name):
+        """Remove a unique Garden
+
+        Args:
+            garden_name (String): Name of Garden to remove
+
+        Returns:
+            bool: True if removal was successful
+
+        Raises:
+            FetchError: Couldn't find a System matching given parameters
+
+        """
+        if self.client.delete_garden(garden_name).ok:
+            return True
+        else:
+            raise FetchError
```

Comment:
I was going to be nitpicky and suggest that you really want to be raising `NotFoundError` here, but actually it turns out that you don't have to worry about any of that.  The `@wrap_response` decorator that you're already using elsewhere handles all of this for you.  So this method ends up looking something like:

```python
@wrap_response(return_boolean=True)
def remove_garden(self, garden_name):
    """"Docstring goes here"""
    return self.client.delete_garden(garden_name)
```

Reply:
I added `raise_404=True` to the wrapper for this func since you said it should raise a `NotFoundError` here.

Reply:
Looking at it now, there are actually a few places where it seems `FetchError` is used in place of raising a `NotFoundError`. `get_systems` does this, for example. 

It's very possible that changing what these functions raise might break integration tests, or something to a similar effect. Considering I don't think it makes any functional difference to change them from `FetchError` to `NotFoundError`, I don't think it's necessary to change them, but it seems important enough to point out.

Reply:
I think that's the default behavior, but being explicit is cool.